src/commands/sbt_cached.yml

```diff
@@ -13,15 +13,19 @@ parameters:
     description: "Save the cache after the command is executed"
     type: boolean
     default: true
+  cache_prefix:
+    description: "The prefix of cache to be used"
+    type: string
+    default: sbt-new-cache
```

Comment:
I'm not a big fan of `sbt-new-cache`. If in the future we want to change it again the `new-cache` will be the old one, which might be confusing. Maybe `sbt-cache-v2` or something like that would be better. WDYT?

Reply:
`sbt-cache-022020` as a proposal, so that we know, more or less how old the cache can be.

Reply:
I'd say to use `year-month` format: (`sbt-cache-2020-2`) so you can use same year's cache if no cache of that month is found.
If the goal is to _not_ use the old cache that's not the best name because the first part will match: `sbt-cache-0`. Isn't it?

Reply:
I think that the semantics that should apply here is: date changed -> invalidate the whole cache.
That's at least the reasoning behind the proposal.